modules/serverless-logic-creating-workflows.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-creating-workflows"]
+= Creating a workflow
+
+You can use the `create` command with `kn workflow` to set up a new {ServerlessLogicProductName} project in your current directory.
+
+.Prerequisites
+
+* The {ServerlessLogicOperatorName} is installed on the cluster.
```

Comment:
{ServerlessLogicOperatorName} will not reflect because I have added this attribute in the (https://github.com/openshift/openshift-docs/pull/76488). Once we merge that PR this attribute will reflect correctly.